src/serac/physics/tests/solid_shape.cpp

```diff
@@ -191,12 +198,10 @@ TEST(SolidMechanics, MoveShapeNonlinear) { shape_test(GeometricNonlinearities::O
 int main(int argc, char* argv[])
 {
   ::testing::InitGoogleTest(&argc, argv);
-  MPI_Init(&argc, &argv);
+  serac::initialize(argc, argv);
 
   axom::slic::SimpleLogger logger;
```

Comment:
After the change to `serac::initialize` you dont need the creation of the `SimpleLogger` since the serac function creates one.